### PR TITLE
aws-help: fix more information link

### PR DESCRIPTION
**Pull request by syleung**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/aws-help.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # aws help
 
 > Display help information about the AWS CLI.
-> More information: <https://docs.aws.amazon.com/cli/latest/reference/>.
+> More information: <https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html>.
```
Comment (mfrw) [inline pages/common/aws-help.md:4]: In my opinion pointing towards reference makes more sense.

Comment (syleung) [reply pages/common/aws-help.md:4]: The original link points to the generic `aws cli` reference:
https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html

But seeing this is specific for the `aws help` command, so I thought the link:
https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html

is more relevant, as the page proves further details on how to use `aws help`.

Comment (marchersimon) [reply pages/common/aws-help.md:4]: I agree with the change. The `help` subcommand isn't even listed in the old link.

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**